### PR TITLE
✨ nftEntity totalEmotes

### LINKED ISSUE #121: sort by most reacted
as discussed in the issue linked below, we lost most reacted sorting option by going from magick to rubick

https://github.com/kodadot/nft-gallery/issues/3707

### PR DESCRIPTION
**Pull request by roiLeo**
**Thank you for your contribution** to the [KodaDot Indexer](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Feature

### What's new?

- [x] PR continue #122 
- [x] PR closes #121 
- [x] This PR is required for kodadot/nft-gallery#1174

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've run the indexer and it hasn't failed
- [x] I've changed schema and performed a migration
- [x] I found edge cases: DB MIGRATION & `Property 'totalEmotes' does not exist on type 'NFTEntity'`

### Screenshot

Capture d’écran 2022-10-13 à 5 11 58 PM

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -48,6 +48,7 @@ type NFTEntity @entity {
   meta: MetadataEntity
   createdAt: DateTime!
   updatedAt: DateTime!
+  totalEmotes: Int!
```
Comment (vikiival) [inline schema.graphql:51]:
```suggestion
  emoteCount: Int!
```

**Review by vikiival — CHANGES_REQUESTED** [review]
just naming

**Comment by vikiival** [comment]
To the naming: I would like to avoid `totalEmotes` as `s` evokes `Array` 
+ usually `s` is also for GraphQL nested entities.

File: schema.graphql (hunk 1 above)
Comment (roiLeo) [reply schema.graphql:51]: Noted! I'll change that

**Comment by roiLeo** [comment]
updated!

Capture d’écran 2022-10-13 à 5 33 07 PM

**Pull request merged**